victkarangwa/WayFarer
# Bg travis status 167918433

#### What does this PR do?
- Resolve Travis build issue
#### Description of Task to be completed?
- Previously, the build was failing to pass due to the DB issue
#### How should this be manually tested?
- Navigate to develop branch of this repo
- Check the build status
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[PT Link](https://www.pivotaltracker.com/story/show/167918433)
#### Screenshots
N/A
#### Questions:
N/A
